packages/docs-site/package.json

```diff
@@ -42,10 +42,13 @@
     "veaury": "^2.3.11"
   },
   "devDependencies": {
-    "@penrose/roger": "^3.0.0-alpha.1",
+    "@penrose/core": "^3.0.0-alpha.0",
     "@penrose/editor": "^3.0.0-alpha.0",
+    "@penrose/roger": "^3.0.0-alpha.1",
+    "@types/markdown-it": "^12.2.3",
+    "markdown-it": "^13.0.1",
     "markdown-it-katex": "^2.0.3",
     "shx": "^0.3.3",
-    "vitepress": "^1.0.0-alpha.35"
+    "vitepress": "^1.0.0-beta.1"
```

Comment:
Ah cool! Did this version bump change anything?

Reply:
Not really. Was attempting this to fix the SSR error in production build. Didn't end up helping but I figured it's better to use the beta version instead of alpha anyway.